### ISSUE TITLE
Question: AI-powered Bing & Bing Search Engine Results API

### ISSUE THREAD

**Issue by jenskastensson** [serpapi/public-roadmap#632]
Interested in anonymous access to the 'Bing Chat' via SerpAPI, any news on this topic would be appreciated
https://blogs.microsoft.com/blog/2023/02/07/reinventing-search-with-a-new-ai-powered-microsoft-bing-and-edge-your-copilot-for-the-web/

**Comment by aliayar** [comment]
Excert from the blog post above:

> Live today in limited preview, expanding to millions soon
> 
> The new [Bing](https://www.bing.com/) is available today in a limited preview on desktop, and everyone can visit [Bing.com](https://www.bing.com/) today to try sample queries and sign up for the waitlist. We’re going to scale the preview to millions in the coming weeks. A mobile experience will also be in preview soon.
> 
> We’re excited to put the new Bing and Edge into the real world to get the critical feedback required to improve our models as we scale.

As soon as the preview is public, we will start working on adding the capabilities into Bing API.

**Comment by kagermanov27** [comment]
Since they are using LLMs in the background in real-time, we would need a constant connection and, worst case scenario, a web driver action. I'm not saying it is a bad idea. I am just pointing out to technical issues we might face.

**Comment by hartator** [comment]
> Since they are using LLMs in the background in real-time

Oh, you're right. I thought Bing was "faking" the typing effect. But, it is streamed live:

![image](https://user-images.githubusercontent.com/307597/218171855-5a0b8b39-5865-4dcc-9486-f9a2e13f4a02.png)

**Comment by kagermanov27** [comment]
>Actually, the network stream might be "faked" as well. Everything still works and loaded fully via the typing visual effect even when I cut the Wifi immediately after page refresh:

We need to make sure it works on scale. The result you got might be cached.

**Comment by hartator** [comment]
> We need to make sure it works on scale. The result you got might be cached.

Yes, and the actual implementation is probably gonna be very different. We have to wait I suppose.

**Comment by ilyazub** [comment]
The New Bing's "AI-powered" right sidebar is in the HTML of the public search results already.

![image](https://github.com/serpapi/public-roadmap/assets/282605/8a4fe249-81d3-4f04-b44f-c46384411d66)

[Search Inspector](https://serpapi.com/searches/6470838dc90e0a5813cef3ac/inspect)

**Comment by hilmanski** [comment]
I'll close this thread as we now support Bing Copilot answers:

Docs: https://serpapi.com/bing-copilot-api
